platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorStyledTextView.kt

```diff
@@ -128,4 +134,38 @@ open class EditorStyledTextView : AppCompatTextView {
             }
         )
     }
+
+    override fun onTouchEvent(event: MotionEvent?): Boolean {
```

Comment:
Please could you also add a UI test for the new behaviour?

Reply:
I'll try to, but it's not that simple to select a pill in the TextView. Maybe if I make the *only* content of the TextView it'll make things easier.

Reply:
No problem - thanks for adding these tests!